app/controllers/assignments_controller.rb

```diff
@@ -38,16 +38,15 @@ def generate_rotation
     start_user = params.require :starting_user_id
     @roster.generate_assignments(user_ids, start_date,
                                  end_date, start_user).each do |assignment|
-      assignment.notify :owner, of: :create, by: @current_user
+      assignment.notify :owner, of: :new_assignment, by: @current_user
     end
     # TODO: undo
     flash[:message] = 'Rotation has been generated.'
     redirect_to roster_assignments_path(@roster, date: start_date)
   end
 
-  # rubocop:disable Metrics/AbcSize, MethodLength
-
   def index
+    # rubocop:disable Metrics/AbcSize, MethodLength
```

Comment:
Would you mind just configuring the abc size globally to be like 5 lines longer? We run into this issue a lot and always ignore it.